tools/wpt/testfiles.py

```diff
@@ -379,7 +380,9 @@ def get_revish(**kwargs):
     revish = kwargs["revish"]
     if revish is None:
         revish = "%s..HEAD" % branch_point()
-    assert isinstance(revish, str)
+        if isinstance(revish, six.text_type):
+            revish = revish.encode("utf8")
+    assert isinstance(revish, six.binary_type)
```

Comment:
If we're asserting this using `six` here, let's change the return type to `bytes`. Some tests would be nice, too.